### PR TITLE
Madland blast

### PR DESCRIPTION
**Pull request by Deeptivarshney**

### CODE REVIEW

File: topics/sequence-analysis/tutorials/ncbi-blast-against-the-madland/tutorial.md [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+    <th>NCBI name</th>
+    <th>Description</th>
+  </tr>
+  <tr>
+    <td>1</td>
+    <td>qseqid</td>
+    <td>Query Seq-id (ID of your sequence)</td>
+  </tr>
+  <tr>
+    <td>2</td>
```
Comment (hexylena) [inline topics/sequence-analysis/tutorials/ncbi-blast-against-the-madland/tutorial.md:97]:
```suggestion
Column | NCBI Name | Description
--- | --- | ---
1 | qseqid | Query Seq-id (ID of your sequence)
```

here you can use a markdown table which we would prefer, if possible.

File: topics/sequence-analysis/tutorials/ncbi-blast-against-the-madland/tutorial.md [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+{: .details}
+
+## More Similarity Search Tools on Galaxy
+
+* **Diamond**: {% tool [Diamond](toolshed.g2.bx.psu.edu/repos/bgruening/diamond/bg_diamond/2.0.15+galaxy0) %} is a high-throughput program for alignment of  large-scale data sets. It aligns sequences to the reference database using a compressed version of the reference sequences called a "database diamond" which is faster to read and can save computational time (~20,000 times the speed of Blastx, with high sensitivity).
+
+> <details-title>Diamond in depth</details-title>
+>
+> [Buchfink et al. (2015): Fast and sensitive protein alignment using Diamond.](https://www.nature.com/nmeth/journal/v12/n1/abs/nmeth.3176.html)
+{: .details}
```
Comment (hexylena) [inline topics/sequence-analysis/tutorials/ncbi-blast-against-the-madland/tutorial.md:171]:
```suggestion
> <details-title>Diamond in depth</details-title>
> See {% cite Buchfink2014 %} for more discussion.
{: .details}
```

here you can use a proper `citation`, just add the following to the `tutorial.bib` file in the same folder as your tutorial.md

```
@article{Buchfink2014,
  doi = {10.1038/nmeth.3176},
  url = {https://doi.org/10.1038/nmeth.3176},
  year = {2014},
  month = nov,
  publisher = {Springer Science and Business Media {LLC}},
  volume = {12},
  number = {1},
  pages = {59--60},
  author = {Benjamin Buchfink and Chao Xie and Daniel H Huson},
  title = {Fast and sensitive protein alignment using {DIAMOND}},
  journal = {Nature Methods}
}
```

File: topics/sequence-analysis/tutorials/ncbi-blast-against-the-madland/tutorial.md [hunk 3]
```diff
@@ -0,0 +1,172 @@
...
+> The fields are separated by tabs, and each row represents a single hit. For more details for BLAST analysis and output, we recommand you to follow the [Similarity-searches-blast](https://training.galaxyproject.org/training-material/topics/genome-annotation/tutorials/genome-annotation/tutorial.html#similarity-searches-blast) tutorial.
+
+
+> <details-title>Further Reading about BLAST Tools in Galaxy</details-title>
+>
+>
+> Cock et al. (2015): [NCBI BLAST+ integrated into Galaxy](http://biorxiv.org/content/early/2015/05/04/014043.full-text.pdf+html)
+>
+> Cock et al. (2013): [Galaxy tools and workflows for sequence analysis with applications in molecular plant pathology](https://peerj.com/articles/167/)
+{: .details}
```
Comment (hexylena) [inline topics/sequence-analysis/tutorials/ncbi-blast-against-the-madland/tutorial.md:162]: If you could replace these with proper citations as well (see comment below) that'd be really helpful!

File: topics/sequence-analysis/tutorials/ncbi-blast-against-the-madland/tutorial.md [hunk 4]
```diff
@@ -0,0 +1,172 @@
...
+
+<!-- This is a comment. -->
+
+MAdLandDB is a protein database comprising of a comprehensive collection of fully sequenced plant and algal genomes, with a particular emphasis on non-seed plants and streptophyte algae. Additionally, for comparative analysis, the database also includes genomes from various other organisms such as fungi, animals, the SAR group, bacteria, and archaea. The database is actively developed and maintained by the [Rensing lab](http://plantco.de) and released in the [MAdLand](madland.science) setting. It employs a system of species abbreviation using a 5 letter code, which is constructed using the first three letters of the genus and the first two letters of the species name, for example, CHABR for *Chara braunii*. Furthermore, the database provides gene identification through the addition of gene ID’s and supplementary information such as the encoding source of the gene, whether it is plastome encoded (pt) or transcriptome-based (tr) in cases when a genome is not yet available. The key advantage of this database is its non-redundant nature, and the fact that all sequences are predominantly from genome projects, thereby increasing their reliability.
```
Comment (hexylena) [inline topics/sequence-analysis/tutorials/ncbi-blast-against-the-madland/tutorial.md:27]:
```suggestion
MAdLandDB is a protein database comprising of a comprehensive collection of fully sequenced plant and algal genomes, with a particular emphasis on non-seed plants and streptophyte algae. Additionally, for comparative analysis, the database also includes genomes from various other organisms such as fungi, animals, the SAR group, bacteria, and archaea. The database is actively developed and maintained by the [Rensing lab](http://plantco.de) and released in the [MAdLand](https://madland.science) setting. It employs a system of species abbreviation using a 5 letter code, which is constructed using the first three letters of the genus and the first two letters of the species name, for example, CHABR for *Chara braunii*. Furthermore, the database provides gene identification through the addition of gene ID’s and supplementary information such as the encoding source of the gene, whether it is plastome encoded (pt) or transcriptome-based (tr) in cases when a genome is not yet available. The key advantage of this database is its non-redundant nature, and the fact that all sequences are predominantly from genome projects, thereby increasing their reliability.
```

**Review by hexylena — COMMENTED** [review]
Some formatting comments :)

File: topics/sequence-analysis/tutorials/ncbi-blast-against-the-madland/tutorial.md (hunk 1 above)
Comment (Deeptivarshney) [reply topics/sequence-analysis/tutorials/ncbi-blast-against-the-madland/tutorial.md]: done :)

File: topics/sequence-analysis/tutorials/ncbi-blast-against-the-madland/tutorial.md (hunk 2 above)
Comment (Deeptivarshney) [reply topics/sequence-analysis/tutorials/ncbi-blast-against-the-madland/tutorial.md]: Thank you for your suggestion, changes have been made accordingly.

File: topics/sequence-analysis/tutorials/ncbi-blast-against-the-madland/tutorial.md (hunk 3 above)
Comment (Deeptivarshney) [reply topics/sequence-analysis/tutorials/ncbi-blast-against-the-madland/tutorial.md]: Changes have been made, Thank you :)

File: topics/sequence-analysis/tutorials/ncbi-blast-against-the-madland/tutorial.md (hunk 2 above)
Comment (Deeptivarshney) [reply topics/sequence-analysis/tutorials/ncbi-blast-against-the-madland/tutorial.md]: Hi @hexylena, 
Thank you for your input. I have made the necessary changes. Can you please take a look and confirm if they are ready to be merged.

**Review by bgruening — APPROVED** [review]
Its still enabled: false, so we will keep iterating.

Thanks @hexylena for your review.

**Pull request merged**